avsomr/avsimport.cpp

```diff
@@ -0,0 +1,120 @@
+
+#include <QFile>
+#include <QByteArray>
+#include <QBuffer>
+
+#include <libmscore/score.h>
+
+#include "avslog.h"
+#include "avsomrreader.h"
+#include "avsomrnetrecognizer.h"
+
+#include <thirdparty/qzip/qzipreader_p.h>
+
+namespace Ms {
+
+extern Score::FileError importMusicXml(MasterScore* score, QIODevice* dev, const QString& name);
+
+namespace {
```

Comment:
`namespace` name?

Reply:
this is an anonymous namespace for the private functions of this file

Reply:
I don't see this kind of operation elsewhere in MuseScore code. Maybe mark those functions as `static` instead?

Reply:
mark static will not allow to avoid name conflicts, anonymous namespace need so that there is no name conflict when linking, if in another .cpp there will be a function with the same name

Reply:
I was mistaken, statics solves this problem, but it’s only deprecated and this is C style, in modern C ++ it is recommended to use anonymous namespace. I will fix it on a static, if so accepted.

Reply:
> in modern C ++ it is recommended to use anonymous namespace

Really? Is there some documentation for this I can read?